pkg/stolon/stolon.go

```diff
@@ -99,6 +99,26 @@ func (c Clusterdata) CheckHealthy() error {
 	return nil
 }
 
+// CheckMinHealthy returns an error if there aren't the minimum number of healthy
+// keepers to perform a failover (healthy keepers >1 excluding the master)
+func (c Clusterdata) CheckMinHealthy() error {
```

Comment:
I think `CheckHealthy` intends to offer what this method does, and we should probably combine the two?

It's meant to be "given our cluster specification, are we healthy?". That means healthy master, appropriate number of standbys.

Reply:
Yeah sure, makes sense!

Reply:
Done!